python/private/pypi/deps.bzl

```diff
@@ -115,7 +116,7 @@ py_library(
         "**/*.dist-info/RECORD",
         "BUILD",
         "WORKSPACE",
-    ]),
+    ] + glob_excludes.version_dependent_exclusions()),
```

Comment:
What about the `whl_library_targets` macro? Should we include the excludes in there too?

Reply:
Yes, we should

Reply:
done